src/AnswerbaseClient.php

```diff
@@ -203,6 +203,14 @@ public function getUserByEmail($email = '')
     /**
      * @codeCoverageIgnore
      */
+    public function getUserByUsername($email = '')
```

Comment:
Shouldn't we call this input `$username` not `$email`? Seems like a clearer variable name.